composer.json

```diff
@@ -11,11 +14,23 @@
         "irc": "irc://irc.freenode.net/yii",
         "source": "https://github.com/yiisoft/yii2-shell"
     },
+    "authors": [
+        {
+            "name": "Daniel Gomez Pan",
+            "email": "pana_1990@hotmail.com"
+        }
+    ],
     "require": {
         "yiisoft/yii2": "~2.0.0",
-        "phpunit/phpunit": "~4.4"
+        "phpunit/phpunit": "~4.4",
+        "psy/psysh": "0.7.*"
```

Comment:
just remove phpunit entirely and leave the rest as is. phpunit is provided by the ci environment if needed.